charts/castai-spot-handler/templates/namespace.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: {{ .Values.namespace }}
```

Comment:
we can remove `Values.namespace` from values, I guess? 
Do we need this namespace resource at all?

Reply:
yes, we can remove, thanks

Do we need this? Not sure. Helm 3 provides `--create-namespace` so in that regard we don't. But we don't pass it in any of our onboarding scripts, so might keep it